build.yaml

```diff
@@ -104,3 +87,23 @@ jobs:
           - template: pipeline/install-node-prerequisites.yaml
           - template: pipeline/electron/electron-e2e-test-interactive.yaml
           - template: pipeline/electron/electron-e2e-publish-results.yaml
+
+    # CI build only
+    - job: 'publish_electron_windows_linux'
+      condition: and(always(), in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI'))
+      pool:
+          vmImage: 'windows-latest'
```

Comment:
nit: Do we want to use a fixed `vmImage` rather than `windows-latest` for this?

Reply:
I could go either way. One thing to keep in mind is that windows agents change very slowly. Here's the list: https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops#use-a-microsoft-hosted-agent A change here would imply a major release.

Reply:
Cool. I dont have a strong preference for either option; just wanted to make sure that we are consistent with our usage of vmImage through out our build pipelines.